.github/workflows/resources-clean.yml

```diff
@@ -35,3 +35,6 @@ jobs:
       
       - name: Clean resources which are created before yesterday
         run: sh tools/workflow/clean-terraform-resources.sh
+
+      - name: Clean distributor documents
+        run: go test e2etest/clean/clean_distributor_test.go -v
```

Comment:
I'm not a fan of using `go test` to invoke functional scripts.  Is there any reason this needs to be done this way rather than having a `package main` and invoking it with `go run`?